core/src/execution/proto/expr.proto

```diff
@@ -219,6 +219,7 @@ message Subtract {
   Expr right = 2;
   bool fail_on_error = 3;
```

Comment:
Instead, set this `fail_on_error` to true when ANSI enabled.

Reply:
Okay. It makes sense & i will refactor, earlier i was referecing `Abs`.